toolbox/library/src/OSQP.cpp

```diff
@@ -338,6 +352,8 @@ bool wbt::block::OSQP::initialize(BlockInformation* blockInfo)
     // PARAMETERS
     // ==========
 
+    int maxIter = 1;
```

Comment:
Maybe initializing to an invalid number would ensure that the parameter is correctly read from the mask

```suggestion
    int maxIter = -1;
```